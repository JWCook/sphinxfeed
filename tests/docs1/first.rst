:date: 2018-09-11

=========
First day
=========

Mul seitse pruuti on,
neid kõiki armastan
ja iga päev neist üht mina külastan.
Ikka Emma esmaspäev, ...
