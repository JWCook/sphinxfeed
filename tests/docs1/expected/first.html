<!DOCTYPE html>

<html lang="en" data-content_root="./">
  <head>
    <meta charset="utf-8" />
    <meta name="viewport" content="width=device-width, initial-scale=1.0" /><meta name="viewport" content="width=device-width, initial-scale=1" />

    <title>First day &#8212; Joe&#39;s website</title>
    <link rel="stylesheet" type="text/css" href="_static/pygments.css?v=fa44fd50" />
    <link rel="stylesheet" type="text/css" href="_static/alabaster.css?v=12dfc556" />
    <script src="_static/documentation_options.js?v=5929fcd5"></script>
    <script src="_static/doctools.js?v=9a2dae69"></script>
    <script src="_static/sphinx_highlight.js?v=dc90522c"></script>
    <link rel="index" title="Index" href="genindex.html" />
    <link rel="search" title="Search" href="search.html" />
    <link rel="next" title="Second day" href="second.html" />
    <link rel="prev" title="Joe’s blog" href="index.html" />
   
  <link rel="stylesheet" href="_static/custom.css" type="text/css" />
  

  
  

  </head><body>
  

    <div class="document">
      <div class="documentwrapper">
        <div class="bodywrapper">
          

          <div class="body" role="main">
            
  <section id="first-day">
<h1>First day<a class="headerlink" href="#first-day" title="Link to this heading">¶</a></h1>
<p>Mul seitse pruuti on,
neid kõiki armastan
ja iga päev neist üht mina külastan.
Ikka Emma esmaspäev, …</p>
</section>


          </div>
          
        </div>
      </div>
      <div class="sphinxsidebar" role="navigation" aria-label="Main">
        <div class="sphinxsidebarwrapper">
<h1 class="logo"><a href="index.html">First sphinxfeed tester</a></h1>








<h3>Navigation</h3>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">First day</a></li>
<li class="toctree-l1"><a class="reference internal" href="second.html">Second day</a></li>
</ul>

<div class="relations">
<h3>Related Topics</h3>
<ul>
  <li><a href="index.html">Documentation overview</a><ul>
      <li>Previous: <a href="index.html" title="previous chapter">Joe’s blog</a></li>
      <li>Next: <a href="second.html" title="next chapter">Second day</a></li>
  </ul></li>
</ul>
</div>
<search id="searchbox" style="display: none" role="search">
  <h3 id="searchlabel">Quick search</h3>
    <div class="searchformwrapper">
    <form class="search" action="search.html" method="get">
      <input type="text" name="q" aria-labelledby="searchlabel" autocomplete="off" autocorrect="off" autocapitalize="off" spellcheck="false"/>
      <input type="submit" value="Go" />
    </form>
    </div>
</search>
<script>document.getElementById('searchbox').style.display = "block"</script>








        </div>
      </div>
      <div class="clearer"></div>
    </div>
    <div class="footer">
      &#169;2018 Joe Doe.
      
      |
      Powered by <a href="https://www.sphinx-doc.org/">Sphinx 7.4.5</a>
      &amp; <a href="https://alabaster.readthedocs.io">Alabaster 0.7.16</a>
      
      |
      <a href="_sources/first.rst.txt"
          rel="nofollow">Page source</a>
    </div>

    

    
  </body>
</html>